dev/devicelab/test/cocoon_test.dart

```diff
@@ -154,52 +155,7 @@ void main() {
           '"ResultData":{"i":0.0,"j":0.0,"not_a_metric":"something"},'
           '"BenchmarkScoreKeys":["i","j"]}';
       fs.file(resultsPath).writeAsStringSync(updateTaskJson);
-      await cocoon.sendResultsPath(resultsPath);
-    });
-
-    test('sends expected request from successful task', () async {
-      mockClient = MockClient((Request request) async => Response('{}', 200));
-
-      cocoon = Cocoon(
-        serviceAccountTokenPath: serviceAccountTokenPath,
-        fs: fs,
-        httpClient: mockClient,
-        requestRetryLimit: 0,
-      );
-
-      final TaskResult result = TaskResult.success(<String, dynamic>{});
-      // This should not throw an error.
-      await cocoon.sendTaskResult(builderName: 'builderAbc', gitBranch: 'branchAbc', result: result);
-    });
-
-    test('throws client exception on non-200 responses', () async {
```

Comment:
The other two tests make sense to remove now, but this one is useful. It validates if there was an issue with the network request the test should fail

Reply:
Added back.